tests/Unit/DataStructures/Tensor/EagerMath/Test_Magnitude.cpp

```diff
@@ -261,6 +267,87 @@ void test_root_tags() {
+          Tags::PointwiseL2Norm<MyScalar>, Tags::PointwiseL2Norm<Vector>,
+          Tags::PointwiseL2Norm<Covector<3>>, Tags::PointwiseL2Norm<Metric>,
+          Tags::PointwiseL2Norm<InverseMetric>, Tags::L2Norm<MyScalar>,
+          Tags::L2Norm<Vector>, Tags::L2Norm<Covector<3>>, Tags::L2Norm<Metric>,
+          Tags::L2Norm<InverseMetric>>>(my_scalar, vec, covec, psi, invpsi);
+
+  const auto local_norm_vec =
+      sqrt(square(get<0>(vec)) + square(get<1>(vec)) + square(get<2>(vec)));
+  const auto local_norm_covec = sqrt(
+      square(get<0>(covec)) + square(get<1>(covec)) + square(get<2>(covec)));
+  const auto local_norm_psi = sqrt(
+      square(get<0, 0>(psi)) + square(get<0, 1>(psi)) + square(get<0, 2>(psi)) +
+      square(get<1, 0>(psi)) + square(get<1, 1>(psi)) + square(get<1, 2>(psi)) +
+      square(get<2, 0>(psi)) + square(get<2, 1>(psi)) + square(get<2, 2>(psi)));
+  const auto local_norm_invpsi =
+      sqrt(square(get<0, 0>(invpsi)) + square(get<0, 1>(invpsi)) +
+           square(get<0, 2>(invpsi)) + square(get<1, 0>(invpsi)) +
+           square(get<1, 1>(invpsi)) + square(get<1, 2>(invpsi)) +
+           square(get<2, 0>(invpsi)) + square(get<2, 1>(invpsi)) +
+           square(get<2, 2>(invpsi)));
+
+  CHECK(get(db::get<Tags::PointwiseL2Norm<MyScalar>>(box)) ==
+        DataVector{npts, 3.0});
+  CHECK(get(db::get<Tags::PointwiseL2Norm<Vector>>(box)) == local_norm_vec);
+  CHECK(get(db::get<Tags::PointwiseL2Norm<Covector<3>>>(box)) ==
+        local_norm_covec);
+  CHECK(get(db::get<Tags::PointwiseL2Norm<Metric>>(box)) == local_norm_psi);
+  CHECK(get(db::get<Tags::PointwiseL2Norm<InverseMetric>>(box)) ==
+        local_norm_invpsi);
+
+  using PlusSquare = funcl::Plus<funcl::Identity, funcl::Square<>>;
+  const double local_acc_norm_scalar =
+      sqrt(alg::accumulate(DataVector{npts, 3.0}, 0., PlusSquare{}) / npts);
+  const double local_acc_norm_vec =
+      sqrt(alg::accumulate(local_norm_vec, 0., PlusSquare{}) / npts);
+  const double local_acc_norm_covec =
+      sqrt(alg::accumulate(local_norm_covec, 0., PlusSquare{}) / npts);
+  const double local_acc_norm_psi =
+      sqrt(alg::accumulate(local_norm_psi, 0., PlusSquare{}) / npts);
+  const double local_acc_norm_invpsi =
+      sqrt(alg::accumulate(local_norm_invpsi, 0., PlusSquare{}) / npts);
+
+  CHECK(db::get<Tags::L2Norm<MyScalar>>(box) == approx(local_acc_norm_scalar));
+  CHECK(db::get<Tags::L2Norm<Vector>>(box) == approx(local_acc_norm_vec));
+  CHECK(db::get<Tags::L2Norm<Covector<3>>>(box) ==
+        approx(local_acc_norm_covec));
+  CHECK(db::get<Tags::L2Norm<Metric>>(box) == approx(local_acc_norm_psi));
+  CHECK(db::get<Tags::L2Norm<InverseMetric>>(box) ==
+        approx(local_acc_norm_invpsi));
+
+  using Tag = MyScalar;
+  /// [l2norm_name]
+  CHECK(Tags::PointwiseL2Norm<Tag>::name() ==
+        "PointwiseL2Norm(" + db::tag_name<Tag>() + ")");
+  CHECK(Tags::L2Norm<Tag>::name() == "L2Norm(" + db::tag_name<Tag>() + ")");
+  /// [l2norm_name]
```

Comment:
I don't quite understand what this snippet is useful for, also in the other tests in this file. Is the idea to print the `name` to the documentation? @sxs-collaboration/spectre-core-devs are we doing this now?

Reply:
Such tests of `SimpleTag`s is quite generic practice in spectre. See for e.g. [here](https://github.com/sxs-collaboration/spectre/blob/develop/tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeSpacetimeQuantities.cpp#L159) or [here](https://github.com/sxs-collaboration/spectre/blob/develop/tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp#L593) or ...